_posts/2018-05-25-support-process.markdown

```diff
@@ -0,0 +1,99 @@
+
+We’ve also seen a couple of team configurations, each with its own effect. When I started at Artsy in 2014, engineers were organized into “practice” teams, meaning the work we did corresponded roughly to the systems we maintained. After that, we organized by “business unit”, meaning there were more teams sharing systems, but non-engineers at Artsy had a clear group of people who they could ask for help. For client-facing systems it was easier to route questions to the right people, but ownership was less clear for internal systems that were shared.
+
+Our ad-hoc support practice was working, but also had some clear drawbacks:
+
+1.  Answering support issues took a lot of time/focus away from engineers working on sprint tasks, and this responsibility was not distributed
+2.  Bugs and maintenance items weren’t always prioritized or worked on consistently.
+3.  Knowledge about how systems work or how decisions were made was siloed in individuals or small teams
+4.  Issues related to features that were not “owned” by an obvious product area often went ignored
+
+We wanted to create a system that would ensure stable, constantly improving services in spite of all of Artsy’s growing and changing.
+
+# Goals/Research
+
+We first identified the goals of our ideal support process:
+
+1.  People who report issues have full confidence that their issue will be solved, or understand why it won’t be.
+2.  Engineers feel empowered and able to fix issues and understand when it is their responsibility to do so.
+3.  We evolve our systems to require less support and spread knowledge among engineers so there are fewer bottlenecks.
+4.  All consumers of Artsy (employees, users, and partners) feel confident that our systems are stable and we will address any issues that arise.
+
+With those goals in mind, we researched both externally (looking at how other companies tackled this common problem) and internally (talking to members of the engineering team and various business/product stakeholders).
+
+Out of this effort, we learned that people had some shared concerns:
+
+* The same few people tended to address most of the support questions, meaning the responsibility was imbalanced.
+* People felt wary about the idea of having to answer to support requests from areas they were unfamiliar with.
+* People wondered how a support duty might fit into their product work.
+* Without a dedicated process people worried that even if bugs were identified they might not be fixed (thus contributing to future support needs).
```

Comment:
Could be rephrased for clarity. Maybe something about a "more consistent and accountable process for triaging and prioritizing bug fixes."